keg/db/dialect_ops.py

```diff
@@ -78,14 +77,7 @@ def drop_all(self):
         self.execute_sql(sql)
 
     def prep_empty(self):
-        sql = []
-        connection_user = self.engine.url.username
-        for schema in self.opt_schemas:
-            sql.extend([
-                'CREATE SCHEMA "{}" AUTHORIZATION "{}";'.format(schema, connection_user),
-                'GRANT ALL ON SCHEMA "{}" TO "{}";'.format(schema, connection_user),
-            ])
-        self.execute_sql(sql)
+        self.create_schemas()
```

Comment:
Now that I'm thinking about it, it may not make sense to create the schemas on empty.  Only on create.  What do you all think?

Reply:
If I'm running db-clear, I'd expect the resulting database to be empty - no tables, no schemas, etc. Creating the schemas should just be the first step in database init, so the rest of the DDL can assume they're present.

Having that step after clearing the database muddies the waters for me: I'm left with a database that's not really clear.

Reply:
That said, @rsyring do we really need a `prep_empty` at all? If we move schema setup to the front of db init, we already have a `db_init_pre` signal if anything needs hooked in. I'd move to deprecate `prep_empty` and make it a no-op by default.

Reply:
When running `develop db init`, we're not guaranteed it's an empty database we're working with, so keeping the empty nomenclature is a bit counterintuitive.

Reply:
I don't remember why I created things this way. :)  I agree that we could probably just get rid of `prep_empty()`.  If we make `db init` idempotent, then regardless of what state the DB is in when I run it, at the very least the objects my app needs are there.  I think that is really the end goal.  So an empty that always does it's thing and an init that always does it's thing should be what we are shooting for.  If you agree, then feel free to make the changes needed to head in that direction.